swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```
@@ -69,10 +88,41 @@ Follow the steps below to create a new role in your AWS account.
+public function greet(lambda:Context ctx, lambda:FunctionURLRequest request) returns json {
+    lambda:FunctionURLResponse response = {
+        statusCode: 201,
+        headers: {"Content-Type": "application/json"},
+        body: string `{"path": "${request.rawPath}"}`
+    };
+    return response.toJson();
+}
+```
+
+>**Note:** The compiler validates the signature of every annotated function and fails the build when it does not match, for example, when the function has no parameters, has defaultable or rest parameters, has a first parameter that is not `lambda:Context`, or returns an unsupported type. Earlier versions excluded such a function from the generated artifacts without reporting anything.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)